Source/WebCore/page/Quirks.cpp

```diff
@@ -1411,13 +1411,13 @@ bool Quirks::allowLayeredFullscreenVideos() const
 #endif
 
 #if PLATFORM(VISION)
-// twitter.com: rdar://124180748
+// x.com: rdar://132850672
 bool Quirks::shouldDisableFullscreenVideoAspectRatioAdaptiveSizing() const
 {
     if (!needsQuirks())
         return false;
 
-    return isDomain("twitter.com"_s);
+    return isDomain("x.com"_s);
```

Comment:
Please add a comment:
// FIXME(rdar://124579556): Remove once x.com adjusts video handling for visionOS

Reply:
@brentfulgham - Done. :-)